english/choose-pill/choose-pill.md

```diff
@@ -0,0 +1,3 @@
+Do you want to learn the truth about this marshmellow world? [Choose the red pill](https://www.youtube.com/watch?v=qHkXFsK6UUg)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.